render/tests/issue-1773.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
Don't add new HTML files for things that could just be automated. These aren't acceptance tests, so they should be put in `test-render.js` within this directory. Refer to the ospec docs within this repo if you aren't familiar with the test framework.

Reply:
I can convert it into tests inside `test-render.js` if the basis of this PR fit specs.

I planned this file to be temporary (was unsure about adding it in a commit or a message on Github) and should be removed before a merge if a revision of this PR might be good.